### PR TITLE
Feature/#102add fake development endpoint 

### PR DESCRIPTION
**Pull request by pupadevs**
#102 
Add fake endpoint for development block

### CODE REVIEW

**Review by AlejandroNPereyra — APPROVED** [review]
Genial!

**Review by coxen34 — APPROVED** [review]
Fantástico!! Muy veloz!!

File: app/Annotations/OpenApi/controllersAnnotations/developmentAnnotations/AnnotationsDevelopmentList.php
Comment (FranEnLaNube) [inline app/Annotations/OpenApi/controllersAnnotations/developmentAnnotations/AnnotationsDevelopmentList.php:1]: I think the path should be without the "api/v1" section. In future PR the complete server is already added. So we'll need to keep an eye on it.

File: routes/api/v1.php [hunk 1]
```diff
@@ -10,6 +11,7 @@
 use App\Http\Controllers\api\StudentDetailController;
 use App\Http\Controllers\api\TagController;
 use App\Http\Controllers\api\SpecializationListController;
+use App\Http\Controllers\api\TagListController;
```
Comment (FranEnLaNube) [inline routes/api/v1.php:14]: Why do you add this?

**Review by FranEnLaNube — COMMENTED** [review]
I know this is already merged, but I wanted to leave wrote that we need to change that URL on the documentation for this endpoint.

**Pull request merged**